### DISCUSSION TITLE
Get to work on Hoobs and HomeKit?

### DISCUSSION THREAD

**Discussion by dayamjz** [mrlt8/docker-wyze-bridge#177] [Q&A]
I got to the point of running the docker-wyze-bridge in the raspberry pi where hoobs is running. I can see the container logs and it connects and gets most cameras. I cannot construct the URLs for rtsp for the Camera FFMPEG plug bridge in Hoobs 4. Even getting the thumbprint or any other way to validate the stream would be helpful. Could it be a problem that hoobs and the wyze container are running in the same machine?

**Comment by mrlt8** [comment]
That shouldn't be an issue.

I did have some trouble with hoobs in the past, but was able to get it working with homebridge.

Have you tried something like this?

```json
{
    "name": "Camera FFmpeg",
    "cameras": [
        {
            "name": "CAM NAME",
            "manufacturer": "WyzeCam",
            "model": "WYZE_CAKP2JFUS",
            "unbridge": true,
            "videoConfig": {
                "source": "-i rtsp://127.0.0.1:8554/cam-name",
                "vcodec": "copy"
            }
        }
    ],
    "platform": "Camera-ffmpeg"
}
```

**Reply by dayamjz** [reply]
Thanks @mrlt8! I got the camera to show up on home kit but the stream does not work. This is the output of the logs from the container:
wyze-bridge    | 🎬 STARTING 1 OF 11 CAMERAS
wyze-bridge    | 2021/10/19 15:51:10 [WyzeBridge] Starting rtsp-simple-server v0.17.3
wyze-bridge    | 2021/10/19 15:51:13 [Kitchen Cam] 🎉 Starting HD 120kb/s Stream for WyzeCam V3 in "LAN mode" FW: 4.36.3.19 IP: 10.0.0.248 WiFi: 100%
wyze-bridge    | 2021/10/19 15:51:18 [RTSP][KITCHEN-CAM] ✅ '/kitchen-cam' stream is UP!
wyze-bridge    | 2021/10/19 15:51:18 [RTSP][KITCHEN-CAM] 📖 New client reading 
wyze-bridge    | 2021/10/19 15:51:24 [RTSP][KITCHEN-CAM] 📕 Client stopped reading
wyze-bridge    | 2021/10/19 15:51:36 [RTSP][KITCHEN-CAM] 📖 New client reading 
wyze-bridge    | 2021/10/19 15:51:38 [RTSP][KITCHEN-CAM] 📕 Client stopped reading
wyze-bridge    | 2021/10/19 15:52:15 [RTSP][KITCHEN-CAM] 📖 New client reading 
wyze-bridge    | 2021/10/19 15:52:18 [RTSP][KITCHEN-CAM] 📕 Client stopped reading
wyze-bridge    | 2021/10/19 15:54:24 [RTSP][KITCHEN-CAM] 📖 New client reading 
wyze-bridge    | 2021/10/19 15:54:30 [RTSP][KITCHEN-CAM] 📕 Client stopped reading

Is there a way to test the stream easily? How can I discover the external url that I can use for the container? I'm pretty sure I have a networking problem of some sort

**Reply by dayamjz** [reply]
Got it to work! Thanks again!
